azure-blob-storage/README.md

```diff
@@ -1,3 +1,6 @@
+#Description
+This plugin verify that there are blobs that was modified in the last *X* days
```

Comment:
Maybe "Check that a given blob contains files that were updated in the last X days"?